src/c++/perf_analyzer/docs/cli.md

```diff
@@ -44,7 +44,9 @@ This document details the Perf Analyzer command line interface:
 ## General Options
 
 #### `-?`
+
```

Comment:
are these intentional?

Reply:
no. that was a change introduced from the editor. removed.